bin/hint_tool/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+edition = "2021"
+name = "examples"
+version = "0.1.0"
+
+[dependencies]
+blockifier = { git = "https://github.com/keep-starknet-strange/blockifier", branch = "snos/callinfo-clone", features = ["clone", "testing"] }
+cairo-vm = { git = "https://github.com/lambdaclass/cairo-vm", features = ["extensive_hints", "cairo-1-hints"] }
+clap = { version = "4.5.4", features = ["derive"] }
+serde = { version = "1.0.188", features = ["derive"] }
+serde_json = { version = "1.0.105", features = ["arbitrary_precision"] }
```

Comment:
Can't you use workspace dependencies for the ones used in the rest of the project? Just to avoid possible conflicts. Not 100% sure how much this makes sense for an example project.